packages/terra-date-picker/src/terra-dev-site/test/date-picker/calendar-filter/CalendarFilterExcludeDates.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import CalendarFilter from '../../../../CalendarFilter';
+import styles from '../common/DatePicker.test.module.scss';
+
+const cx = classNames.bind(styles);
+
+const CalendarFilterExcludeDates = () => (
+  <div className={cx('content-wrapper')}>
+    <h3>Excluded dates : 2020-04-01, 2020-04-03</h3>
+    <CalendarFilter
+      excludeDates={['2020-04-01', '2020-04-03']}
+      selectedDate="2020-04-02"
+    />
```

Comment:
If this is rendered in a div next to other content, how does this appear? Would consumes place this in a flex box to place user inputs/info next to the calendar? I see we are adding in test styles. I know there's been instances were adding these styles "hide" visual bugs with padding/margins.

Reply:
we have an test file where calendar-filter is placed next to buttons inside div : https://terra-framew-inline-dat-vvsq9a.herokuapp.com/tests/terra-date-picker/date-picker/calendar-filter/calendar-filter-controlled  and it seems to display as expected.